### DISCUSSION TITLE
add tasks to Google or Outlook calendar

### DISCUSSION THREAD

**Discussion by siiianew** [InteractionDesignFoundation/add-event-to-calendar-docs#55] [General]
Is it possible to add tasks to Google or Outlook calendar？

**Comment by alies-dev** [comment]
Late answer, for anyone finding this thread.

**Google**: partly. Its deep link parser reads `taskId`, `taskBundleId` and `ddl` (deadline), and these switch the editor into task mode instead of event mode. They reference an existing task rather than creating a new one from a title, so there is no real "create a task from a URL" equivalent to the event parameters.

**Outlook**: no. Its compose deep link is parsed on the server, so there is no client side table to read, and replaying plausible task parameter names against the live form produced nothing.

`taskId`, `taskBundleId` and `ddl` are listed in `services/google.md`.
